### PR TITLE
.travis.yml: allow failures for go tip

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM but something else seems to be failing :thinking:

**Comment by mrunalp** [comment]
restarted the failing jobs.

File: .travis.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ jobs:
       go: "1.10.x"
   allow_failures:
     - os: osx
+    - go: tip
```
[the remark this answers is not in the document]
Comment (giuseppe) [reply .travis.yml]: yes, good point.  I've pushed a new version

**Comment by runcom** [comment]
LGTM

**Pull request merged**